scripts/test-docker

```diff
@@ -17,9 +17,9 @@ test_docker_versions["ubuntu:16.04"]="^18.06|^18.09|^19.03|^20.10.[2-7]"
 test_docker_versions["ubuntu:18.04"]="^18.06|^18.09|^19.03|^20.10"
 test_docker_versions["ubuntu:20.04"]="^19.03.9$|^19.03.1[0-9]|^20.10"
 test_docker_versions["centos:centos7"]="^18.06|^18.09|^19.03|^20.10"
-test_docker_versions["centos:centos8"]="^20.10"
 test_docker_versions["oraclelinux:7"]="^19.03|^20.10"
 test_docker_versions["oraclelinux:8"]="^20.10"
+test_docker_versions["rockylinux:8"]="^20.10"
```

Comment:
You have two options to fix the failing CI.
1. Add `test_docker_versions["rockylinux:8"]="20.10.12"` here because rockylinux support is not there in other versions.
Or
2. Add `rockylinux` support for all the 20.10.* versions. (PS. I added rockylinux support on `20.10.8`, `20.10.9`, `20.10.10` and `20.10.11` on this PR #91 )